app/experimenter/nimbus-ui/src/components/AppLayoutWithExperiment/index.tsx

```diff
@@ -88,7 +81,7 @@ const AppLayoutWithExperiment = ({
     };
   }, [startPolling, stopPolling, experiment, polling]);
 
-  if (experimentLoading || analysisRequired) {
```

Comment:
The issue, I noticed, was that `experimentLoading` would become false but `analysisRequired` would also become false due to the `useEffect()` that I removed. There, the experiment would be done loading but `analysisLoading` would still be false because it would not have started loading yet. Then `analysisRequired` is set to false too soon, causing the summary table to display first.

I found that `analysisRequired` didn't seem necessary since we can just check `analysisLoading` instead.